README.md

```diff
@@ -58,50 +58,72 @@ Running the integration tests locally requires a message bus that supports AMQP
 
-# Stop the service and remove Docker volumes and networks created by the start script
-scripts/stop
+The underlying `docker-compose up/down` commands can be customised by appending `-- [DOCKER_COMPOSE_ARGS]` after any other arguments to the `start/stop` scripts. For example:
+
+```
+# Start the service without attaching to logs
+scripts/start -- --detach
 ```
 
-## Connect to sibling services
+## Test the service
 
-To test this service in combination with other parts of the FFC demo application, it is necessary to connect each service to an external Docker network and shared dependencies, such as message queues. Start the shared dependencies from the [`ffc-demo-development`](https://github.com/DEFRA/ffc-demo-development) repository and then use the `connected-` [`scripts`](./scripts/) to start this service. Follow instructions in other repositories to connect each service to the shared dependencies and network.
+This service reacts to messages retrieved from an AMQP 1.0 message broker so manual testing involves pushing messages into the appropriate message queue. The [start](./scripts/start) script runs [ActiveMQ Artemis](https://activemq.apache.org/components/artemis) alongside the application to provide the required message bus and broker.
+
+The [send-test-mesage](./scripts/send-test-message) script will push a valid message into one of the Artemis queues which the app subscribes to. The container logs should show the message being picked up and processed automatically when the following command is executed.
 
 ```
-# Start the service
-script/connected-start
+# Send a sample message to the Artemis message queue
+scripts/send-test-message
+```
+
+For more detailed testing, messages can be pushed into queues via the Artemis console UI hosted at http://localhost:8161/console/login (username: artemis, password: artemis). Messages should match the format of the sample JSON below.
 
-# Stop the service
-script/connected-stop
 ```
+{
+  "claimId": "MINE123",
+  "propertyType": "business",
+  "accessible": false,
+  "dateOfSubsidence": "2019-07-26T09:54:19.622Z",
+  "mineType": ["gold"],
+  "email": "test@email.com"
+}
+```
+
+## Link to sibling services
+
+To test interactions with sibling services in the FFC demo application, it is necessary to connect each service to an external Docker network, along with shared dependencies such as message queues. The most convenient approach for this is to start the entire application stack from the [`mine-support-development`](https://github.com/DEFRA/mine-support-development) repository.

```

Comment:
This is now ffc-demo-development https://github.com/DEFRA/ffc-demo-development

Reply:
Corrected, thanks